web-app/packages/lib/src/assets/sass/themes/mm-theme-light/variables/_general.scss

```diff
@@ -32,7 +32,7 @@ $highlightFocusBg: rgba($primaryColor, .24) !default;
 
 /// Radius of the corners
 /// @group general
-$borderRadius: 4px;
+$borderRadius: 6px;
```

Comment:
@harminius which components are affected by this change? Can you review all theese visually? Because this is global border radius

Reply:
I tried my best and didn't find any button/tags visual changes.

I noticed I have locally slightly different font but it's not connected with this issue, I assume.

Reply:
We checked with Marcel that PMessage as well as Input fields uses own classes with their borderRadius